src/main/java/teammates/common/datatransfer/attributes/AdminEmailAttributes.java

```diff
@@ -96,7 +96,6 @@ public String getJsonString() {
     @Override
     public void sanitizeForSaving() {
         this.subject = SanitizationHelper.sanitizeTextField(subject);
```

Comment:
Hmm.. I didn't see any reason to remove it since it just removes leading or trailing spaces.